packages/grid/helm/syft/values.yaml

```diff
@@ -24,9 +24,9 @@ mongo:
   # Mongo secret name. Override this if you want to use a self-managed secret.
   secretKeyName: mongo-secret
 
-  # Dev mode default passwords
-  defaultSecret:
-    rootPassword: example
+  # custom secret values
+  secret:
+    rootPassword: null
```

Comment:
Charts will no longer ship with a dev default value. It must be explicitly provided by the cluster owner if `randomizedSecrets=false` else helm will error out because of [this line](https://github.com/OpenMined/PySyft/pull/8598/files#diff-7b312d1bb67bfa5583d9e85276c7b51439bf9f0e19e4f7e4b69f38d53759e6f1R57)